### PR TITLE
Issue 3.chia

### PR DESCRIPTION
**Pull request by mahchiahui**
# Pull request questions

## Which issue does this address

Issue number: #3 

## Why was change needed

initial functionality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+
+None.
+
+Example Playbook
+----------------
+
+    - hosts: senzing
+      vars:
+        senzing_g2_release_version: 2.0.0-20197
+        senzing_eula: I_ACCEPT_THE_SENZING_EULA
```
Comment (docktermj) [inline README.md:30]: We're trying to minimize the number of places where `I_ACCEPT_THE_SENZING_EULA` is published.  Instead can you use a technique similar to what is seen at https://github.com/Senzing/docker-yum#eula

File: molecule/default/converge.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+- name: Converge
+  vars:
+    senzing_eula: I_ACCEPT_THE_SENZING_EULA
```
Comment (docktermj) [inline molecule/default/converge.yml:4]: We're going to have to think this one through.  Let me "sleep on it".   However, I won't block the merge into master over this one.

File: tasks/senzing-project.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+- name: Create Senzing project directory
+  shell:
+    cmd: "python3 /opt/senzing/g2/python/G2CreateProject.py {{ senzing_project_dir }}"
```
Comment (docktermj) [inline tasks/senzing-project.yml:4]: In this ansible role,  senzing.senzingapi,   I don't want to introduce the notion of the G2CreateProject.   We may want to separate that into a separate role in the future.   This role should only do the equivalent of yum/apt install.

File: README.md (hunk 1 above)
Comment (mahchiahui) [reply README.md:30]: hmmm let me see if I can find a workaround for these.

**Review by docktermj — APPROVED** [review]

**Pull request merged**